src/test/java/com/amazon/ion/conformance/expectations.kt

```diff
@@ -228,8 +229,8 @@ private fun TestCaseSupport.denotesSymtok(expectation: AnyElement, actual: Symbo
 }
 
 private fun TestCaseSupport.denotesCodepoints(expectation: SeqElement, actual: String) {
-    val expectedCodePoints = expectation.tail.map { it.longValue }
-    val actualCodePoints = actual.codePoints().toList()
+    val expectedCodePoints: List<Int> = expectation.tail.map { it.longValue.toInt() }
+    val actualCodePoints: List<Int> = actual.codePoints().toList()
     assertEquals(expectedCodePoints, actualCodePoints, createFailureMessage(expectation))
```

Comment:
🗺️ This was always failing because (non-empty) `List<Long>` and `List<Int>` were never equal.